File: generated-sources/scala-finch/mojang-api/src/main/scala/com/github/asyncmc/mojang/api/scala/finch/model/SecurityAnswer.scala
package com.github.asyncmc.mojang.api.scala.finch.model

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import com.github.asyncmc.mojang.api.scala.finch.model.SecurityAnswerId

/**
 * A security question answer given by the player using the id given by Mojang
 * @param id A number giver by Mojang to reply a question
 */
case class SecurityAnswer(id: Int
                )

object SecurityAnswer {
    /**
     * Creates the codec for converting SecurityAnswer from and to JSON.
     */
    implicit val decoder: Decoder[SecurityAnswer] = deriveDecoder
    implicit val encoder: ObjectEncoder[SecurityAnswer] = deriveEncoder
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/org/openapitools/model/SecurityAnswer.kt
package org.openapitools.model

import java.util.Objects
import org.openapitools.model.SecurityAnswerId
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * A security question answer given by the player using the id given by Mojang
 */
data class SecurityAnswer (

        @get:NotNull 
        @JsonProperty("id") override val id: Int
) : SecurityAnswerId{

}

